Makefile

```diff
@@ -65,11 +65,10 @@ docs: clean-docs
 	sphinx-apidoc -d 10 -H "Python Reference" -o docs/py_modules/ kolibri kolibri/test kolibri/deployment/ kolibri/dist/
 	$(MAKE) -C docs html
 
-release: clean assets
-	python setup.py sdist upload
-	python setup.py bdist_wheel upload
-	@echo ""
-	@echo "Now run something like twine -s dist/* to upload all results in the dist/ folder"
+release:
+	ls -l dist/
+	read "\nDo you want to upload everything in dist/*?\n\n CTRL+C to exit."
+	twine upload -s dist/*
```

Comment:
cc @aronasorman just want to make sure you're aware of these changes in case if affects buildkite in any way

Reply:
Thanks, don't think it should affect buildkite -- we don't upload anything through there right now.

Reply:
I think releases on PyPI will continue to be a manual thing

Reply:
`twine` is the command that releases stuff to PyPI and `-s` means to PGP sign the release.